CONTRIBUTING.md

```diff
@@ -94,30 +94,58 @@ request ID to the entry you added to `CHANGELOG.md`.
 
 ### How to Get PRs Merged
 
-A PR is considered to be **ready to merge** when:
-
-* It has received two approvals from Collaborators/Maintainers (at
-  different companies). This is not enforced through technical means
-  and a PR may be **ready to merge** with a single approval if the change
-  and its approach have been discussed and consensus reached.
-* Feedback has been addressed.
-* Any substantive changes to your PR will require that you clear any prior
-  Approval reviews, this includes changes resulting from other feedback. Unless
-  the approver explicitly stated that their approval will persist across
-  changes it should be assumed that the PR needs their review again. Other
-  project members (e.g. approvers, maintainers) can help with this if there are
-  any questions or if you forget to clear reviews.
-* It has been open for review for at least one working day. This gives
-  people reasonable time to review.
-* Trivial changes (typo, cosmetic, doc, etc.) do not have to wait for
-  one day and may be merged with a single Maintainer's approval.
-* `CHANGELOG.md` has been updated to reflect what has been
-  added, changed, removed, or fixed.
-* `README.md` has been updated if necessary.
-* Urgent fix can take exception as long as it has been actively
-  communicated.
-
-Any Maintainer can merge the PR once it is **ready to merge**.
+A PR is considered **ready to merge** when:
+
+* It has received two qualified approvals[^1].
+
+  This is not enforced through automation, but needs to be validated by the
+  maintainer merging.
+  * The qualified approvals need to be from [Approver]s/[Maintainer]s
+    affiliated with different companies. Two qualified approvals from
+    [Approver]s or [Maintainer]s affiliated with the same company counts as a
+    single qualified approval.
+  * PRs introducing changes that have already been discussed and consensus
+    reached only need one qualified approval. The discussion and resolution
+    needs to be linked to the PR.
+  * Trivial changes[^2] only need one qualified approval.
+
+* All feedback has been addressed.
+  * All PR comments and suggestions are resolved.
```

Comment:
I don't think this is currently enforced through technical means but can be.  Should we ask for this to be enabled?

Reply:
Is there a way to override for the exceptions listed below? IIRC the number of required approved wasn't ignorable.

Reply:
I guess that @Aneurysm9 is only asking about getting rid of the following line:

>    * All PR comments and suggestions are resolved.

Reply:
Oh gotcha, SGTM 👍

Reply:
Do we just need an issue to track this?

Reply:
I don't think we need to get rid of that line, I'm asking whether we should enable GitHub's enforcement of that requirement.  I think the requirement to resolve all comments doesn't need exceptions in its enforcement mechanism as a maintainer ready to merge a PR can always resolve the outstanding comments if they feel that an exception is warranted.  It adds work, but not a requirement for an additional person to interact as requiring 2 approvals and clearing approvals on changes would.

Reply:
> I'm asking whether we should enable GitHub's enforcement of that requirement.

Sounds good to me :+1:

Reply:
https://github.com/open-telemetry/community/issues/1388